tests/test_pyglmnet.py

```diff
@@ -324,6 +325,13 @@ def rmse(a, b):
     y_pg = glm.predict(X)
     assert abs(rmse(Y, y_sk) - rmse(Y, y_pg)) < 1.0
 
+    glm = GLM(distr='gaussian', alpha=l1_ratio, reg_lambda=alpha,
```

Comment:
what about `max_iter=5`: we're just testing the specific `fit_intercept` behavior right?